docs/source/changelog.rst

```diff
@@ -6,7 +6,7 @@ Jupyter notebook changelog
 A summary of changes in the Jupyter notebook.
 For more detailed information, see `GitHub <https://github.com/jupyter/notebook>`__.
 
-.. _release-4.1.x:
```

Comment:
@takluyver We certainly could leave 4.1.x. I'm cool to do whatever you and @Carreau think is best  since you have a better handle on frequency of releases and schedules.